### PR TITLE
Change MurmurHash-net-core's TargetFramework to netstandard1.4

### PR DESCRIPTION
**Pull request by SharpSeeEr**
Target netstandard1.4 for greater compatibility.

### CODE REVIEW

**Comment by darrenkopp** [comment]
I'm not really an expert in the net .net standard stuff. Can you summarize what the effect of this change would have on people using .net standard 1.6 and 2.0?

**Comment by SharpSeeEr** [comment]
The netstandard versions are backwards compatible. So if I create a project that targets 1.6 or 2.0 I can reference any libraries that target 1.6/2.0 or below. When building a library is best to target the lowest version you can, which makes it available to more developers.
